src/Symfony/Component/HttpKernel/Tests/Profiler/ProfilerTest.php

```diff
@@ -59,6 +59,13 @@ public function testFindWorksWithInvalidDates()
         $this->assertCount(0, $profiler->find(null, null, null, null, 'some string', ''));
     }
 
+    public function testFindWorksWithStatusCode()
+    {
+        $profiler = new Profiler($this->storage);
+
+        $this->assertCount(0, $profiler->find(null, null, null, null, null, null, 204));
```

Comment:
AFAIU, you do not test anything here. Can you add a test where the count is non-zero?